source/common/router/rds_impl.cc

```diff
@@ -120,21 +123,34 @@ void RdsRouteConfigProviderImpl::registerInitTarget(Init::Manager& init_manager)
   init_manager.registerTarget(*this);
 }
 
+std::string RdsRouteConfigProviderImpl::configAsString() const {
+  std::string json_string;
+  Protobuf::util::MessageToJsonString(route_config_proto_, &json_string);
```

Comment:
made a utility function to stop adding this options at all uses. Changed the other call site to use it as well.